import

```diff
@@ -500,14 +565,17 @@ if (options.recordType === 'users') {
   importer = importPlace;
 } else if (options.recordType === 'places-update') {
   importer = updatePlace;
+} else if (options.recordType === 'records') {
+  importer = importRecord;
 }
```

Comment:
Yeah we throw an error if we don't recognized the type when we validate the command line options. 
```
    throw new Error(`Record type not supported: ${options.recordType}`);
```